src/main/webapp/WEB-INF/web.xml

```diff
@@ -1,24 +1,96 @@
-  </servlet>
-  <servlet-mapping>
-   <servlet-name>gromodify</servlet-name>
-   <url-pattern>/gromodify</url-pattern>
-  </servlet-mapping>
-</web-app>
\ No newline at end of file
+<web-app xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns="http://xmlns.jcp.org/xml/ns/javaee"
+    xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/web-app_4_0.xsd"
+    id="WebApp_ID" version="4.0">
+    <display-name>workshop-servlet-khadri-mart</display-name>
+    <welcome-file-list>
+        <welcome-file>main.jsp</welcome-file>
+    </welcome-file-list>
+    <servlet>
+        <servlet-name>cloadd</servlet-name>
+        <servlet-class>com.khadri.mart.clothes.servlet.AddClothesServlet</servlet-class>
+    </servlet>
+    <servlet-mapping>
+        <servlet-name>cloadd</servlet-name>
+        <url-pattern>/cloadd</url-pattern>
+    </servlet-mapping>
+    <servlet>
+        <servlet-name>clothesmodifyservlet</servlet-name>
+        <servlet-class>com.khadri.mart.clothes.servlet.ModifyClothesServlet</servlet-class>
+    </servlet>
+    <servlet-mapping>
+        <servlet-name>clothesmodifyservlet</servlet-name>
+        <url-pattern>/clothesmodify</url-pattern>
+    </servlet-mapping>
+    <servlet>
+		<servlet-name>clodelete</servlet-name>
+		<servlet-class>com.khadri.mart.clothes.servlet.DeleteClothesServlet</servlet-class>
+	</servlet>
+	<servlet-mapping>
+		<servlet-name>clodelete</servlet-name>
+		<url-pattern>/clodelete</url-pattern>
+    </servlet-mapping>
+    <servlet>
+        <servlet-name>clothesaddJsp</servlet-name>
+        <jsp-file>/clothes_add.jsp</jsp-file>
+    </servlet>
+    <servlet-mapping>
+        <servlet-name>clothesaddJsp</servlet-name>
+        <url-pattern>/clothes_add</url-pattern>
+    </servlet-mapping>
+     <servlet>
+        <servlet-name>clothesmodifyJsp</servlet-name>
+        <jsp-file>/clothes_modify.jsp</jsp-file>
+    </servlet>
+    <servlet-mapping>
+        <servlet-name>clothesmodifyJsp</servlet-name>
+        <url-pattern>/clothes_modify</url-pattern>
+    </servlet-mapping>
+     <servlet>
+        <servlet-name>clothesviewJsp</servlet-name>
+        <jsp-file>/clothes_view.jsp</jsp-file>
+    </servlet>
+    <servlet-mapping>
+        <servlet-name>clothesviewJsp</servlet-name>
+        <url-pattern>/clothes_view</url-pattern>
+    </servlet-mapping>
+    <servlet>
+        <servlet-name>clothesviewallJsp</servlet-name>
+        <jsp-file>/clothes_viewAll.jsp</jsp-file>
+    </servlet>
+    <servlet-mapping>
+        <servlet-name>clothesviewallJsp</servlet-name>
+        <url-pattern>/clothes_viewAll</url-pattern>
+    </servlet-mapping>
+      <servlet>
+		<servlet-name>deletepageJsp</servlet-name>
+		<jsp-file>/clothes_delete_page.jsp</jsp-file>
+		</servlet>
+	<servlet-mapping>
+		<servlet-name>deletepageJsp</servlet-name>
+		<url-pattern>/clothes_delete_page</url-pattern>
+	</servlet-mapping>
+    <servlet>
+		<servlet-name>jspDelete</servlet-name>
```

Comment:
change  jspdelete to deletejsp or  clodeletejsp

Reply:
okay